README.md

```diff
@@ -39,8 +39,8 @@ metrics  | Required. A list of CloudWatch metrics to retrieve and export
 aws_namespace  | Required. Namespace of the CloudWatch metric.
 aws_metric_name  | Required. Metric name of the CloudWatch metric.
 aws_dimensions | Optional. Which dimension to fan out over.
-aws_dimension_select | Optional. Which dimension values to filter. Specify a json object with the dimension name as key and an array of dimension values as value.
-aws_dimension_select_regex | Optional. Which dimension values to filter on with a regular expressoin. Specify a json object with the dimension name as key and an array of regexes that will be applied to the dimension value as value.
+aws_dimension_select | Optional. Which dimension values to filter. Specify a list of the dimension name(s) to filter on, with the selected values as a sub-list.
```

Comment:
It's a map with a list value, so I'd say something like A map from the dimension name to a list of values to select from that dimension.